apis/addonconfigs/cni/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,7 +1,7 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-// Copyright 2022 VMware, Inc. All Rights Reserved.
+// Copyright 2023 VMware, Inc. All Rights Reserved.
```

Comment:
question: this seems updated the year in generated file only, do we need to update the source file as well ?

Reply:
I think doing an audit of copyright headers in all the files and fixing it should be the right thing to do. That seems like a broader exercise.

The goal of this PR is to unblock the CI task `make verify` which unblocks all the PRs.

Reply:
I have created this issue: https://github.com/vmware-tanzu/tanzu-framework/issues/4227 to address fixing copyright headers in all source files. 

Resolving this comment now to unblock CI pipelines and developer PRs